src/CommandingOfficers/Modifiers/UnitInstanceFilter.java

```diff
@@ -0,0 +1,27 @@
+package CommandingOfficers.Modifiers;
```

Comment:
It seems strange to have this and other filters in the `CommandingOfficers` tree when everything it's related to is in the `UnitMods` tree.

Reply:
Sorta, yeah. I'm not convinced I did a very good job of putting files in the right places in this PR, lol.